api/src/domain/user/fetchMunicipalityByName.test.ts

```diff
@@ -0,0 +1,23 @@
+import { fetchMunicipalityByName } from "./fetchMunicipalityByName";
+
+describe("fetchMunicipalityByName", () => {
+  it("returns a record when a valid name is supplied", async () => {
+    const record = await fetchMunicipalityByName("TRENTON");
+    expect(record.townName).toEqual("Trenton");
+  });
+
+  it("returns a empty object when an invalid name is supplied", async () => {
```

Comment:
[sand] On the same note as before - I know this is existing behavior but if these don't match, do we really want to save a user's municipality as empty strings? Should it return undefined instead?